framework/uicomponents/qml/MuseScore/UiComponents/StyledTabButton.qml

```diff
@@ -0,0 +1,61 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import MuseScore.UiComponents 1.0
+
+TabButton {
+    id: root
+
+    property int sideMargin: 0
```

Comment:
Can we remove this?

Reply:
No, by default, the buttons in the toolBar are positioned without spacing. This is an alternative to spacing.